create.go

```diff
@@ -7,8 +7,24 @@ import (
 )
 
 // Create writes a new blank migration file.
-func Create(db *sql.DB, dir, name, migrationType string) error {
-	path, err := CreateMigration(name, migrationType, dir, time.Now())
+func Create(db *sql.DB, dir, name, migrationType string, sequential bool) error {
+	var version string
+	if !sequential {
+		version = time.Now().Format("20060102150405")
```

Comment:
I see, what you just explained makes sense. However, the above line confused me. We shouldn't ever suggest version based on a time date.